docs/source/using/addons.adoc

```diff
@@ -241,6 +241,31 @@ You should at least see the *anyuid* add-on listed.
 This is an important add-on that allows you to run images that do not use a pre-allocated UID.
 By default, this is not allowed in OpenShift.
 
+[[table-list-of-default-addons]]
+.List of default add-ons with description
+
+[cols="1,3"]
+|===
+|Add-on Name    | Description
+
+| anyuid
+| Changes the default security context constraints to allow pods to run with arbitrary UID.
+
+| admin-user
+| Creates a user named 'admin' and assign the cluster-admin role to it.
+
+| registry-route
+| Creates an edge terminated route for the OpenShift registry.
+
+| xpass
+| Imports xPaaS templates.
+|===
+
+[[addons-by-community]]
+== {project} Add-ons by the community
+
+The {project} community has a number of add-ons in the {project}/addons repo. These addons are community supported.
```

Comment:
@anjannath , Link to README of minishift-addons is missing here I guess.

Reply:
@budhrg Yes, i am still working on this part, will update it soon.